sql/src/main/java/io/crate/metadata/cluster/AlterTableClusterStateExecutor.java

```diff
@@ -243,27 +227,27 @@ private ClusterState updateSettings(final ClusterState currentState, final Setti
         }
 
         ClusterBlocks.Builder blocks = ClusterBlocks.builder().blocks(currentState.blocks());
-        maybeUpdateClusterBlock(actualIndices,
+        MetaDataUpdateSettingsService.maybeUpdateClusterBlock(actualIndices,
```

Comment:
what about importing this method instead? now the formatting of the following lines is a bit broken...